oscap_report/html_report/templates/js/interactive_script.js

```diff
@@ -22,7 +24,7 @@ function search_rule(self) { // eslint-disable-line no-unused-vars
         var is_matched_title = $(this).attr("title").toLowerCase().includes(value);
         var is_matched_rule_id = $(this).attr("rule-id").toLowerCase().includes(value);
 
-        var result_of_rule = $(this).attr('result').toLowerCase();
+        var result_of_rule = $(this).attr('result').replace(/ /ug, "-").toLowerCase();
```

Comment:
I'd prefer to have a function that does such kind of id-lization of a string. With such a string you could always be sure that all ids across the code base would look the same, and if the original was the same so would be corresponding ids.

Reply:
I can be a JQuery function, like `$(this).attr('result').as_id_function()`